src/parser/priest/discipline/normalizers/AtonementSuccessiveDamage.ts

```diff
@@ -51,4 +51,13 @@ class AtonementSuccessiveDamage extends EventsNormalizer {
     return fixedEvents;
   }
 }
+
+interface ModifiedDamageEvent extends DamageEvent {
+  __modified: boolean
```

Comment:
__modified should be added to the base Event interface (see below comment)

Reply:
Fixed!